src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
It is already possible to implement pseudo-robust notifications with the current ledger using some retry logic if a notification is missed.